tests/echo-server.php

```diff
@@ -31,6 +31,11 @@ public function onError(\Nekland\Woketo\Exception\WebsocketException $e, \Neklan
     {
         echo '(' . get_class($e) . ') ' . $e->getMessage() . "\n";
     }
+
+    public function onDisconnect(\Nekland\Woketo\Core\AbstractConnection $connection)
+    {
+        echo 'disconnected' . PHP_EOL;
```

Comment:
I believe there's a lot of disconnections with wstest running. (like, really a lot) So I guess you should do nothing here.